content/en-us/mine/prb2/2.2-release-note.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Runtime Bridge 2.2 Release Note"
+weight: 2005
+menu:
+  mine:
+    parent: "mine-prb2"
+---
+
+> If you are for a step-by-step guide, please refer to the [upgrading guide from the community](/en-us/mine/prb2/migrating-from-v2.0-and-v2.1-to-v2.2/).
+
+## Overview
+
+Runtime Bridge 2 uses P2P technologies to improve the mining experience. It allows multiple lifecycle instance share data providers to reduce storage pressure and support data provider redundancy to ensure overall stability.
+
+Deployment guide: https://github.com/Phala-Network/runtime-bridge/wiki/Deployment-Guide-for-Runtime-Bridge-2
+Migrate from v0: https://github.com/Phala-Network/runtime-bridge/wiki/Migrate-from-v0-to-v2
+
+To use with docker: `docker pull phalanetwork/prb:next`
+To use monitor with docker: `docker pull phalanetwork/prb-monitor:next`
+
+## Overview
+
+Version 2.2.0 increases worker synchronization speed hugely and introduces TCP keepalive to improve overall stability.
+
+## Important Message
+> `pRuntime` on workers must be upgraded to version 0.2.4(`phalanetwork/phala-pruntime:22051201`) or greater to work with Runtime Bridge v2.2.0.
+
+## Data Provider
+
+- Re-running the [data provider database migration](https://wiki.phala.network/en-us/mine/prb2/migrating-from-v0/#migrate-data-providerfetch-data) script is required since data needs to be recomputed to work with the improved synching mechanism.
```

Comment:
```suggestion
- Re-running the [data provider database migration](/en-us/mine/prb2/migrating-from-v0/#migrate-data-providerfetch-data) script is required since data needs to be recomputed to work with the improved syncing mechanism.
```